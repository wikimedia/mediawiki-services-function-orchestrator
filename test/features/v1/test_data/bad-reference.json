{
	"Z1K1": "Z7",
	"Z7K1": "Z100101"
}
