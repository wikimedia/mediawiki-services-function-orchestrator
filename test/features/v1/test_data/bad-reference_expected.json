{
	"Z1K1": "Z5",
	"Z5K1": {
		"Z1K1": "Z507",
		"Z507K1": "Could not dereference Z7K1"
	}
}
